src/MizarData/MizarPairedDataTest.cpp

```diff
@@ -156,6 +159,38 @@ const std::vector<const orbit_client_protos::TimerInfo*> kTimerPtrs = [] {
 
 constexpr uint64_t kSamplingPeriod = 10;
 
+template <typename Container, typename AnotherContainer>
+auto MakeMap(const Container& keys, const AnotherContainer& values) {
+  using K = typename Container::value_type;
+  using V = typename AnotherContainer::value_type;
+  using std::begin;
```

Comment:
Why this using declaration?

Reply:
It's ADL dance. It makes the code more generic.

Reply:
This is to bring `begin` and `end` from the `std` namespace into the considered overload set.

This is needed for two reasons:

- When `Container` is a C-style array for example which is living in the global namespace, but their corresponding begin and end overloads are living in a different namespace - (`std`) in this case.
- When a type has no free `begin` / `end` functions defined - only member function begin/end - then `std::begin` would redirect to `container.begin/end`.